docs/docsite/rst/community/committer_guidelines.rst

```diff
@@ -4,50 +4,58 @@
+Any other new features and changes to high level design should go through the proposal process (TBD), to ensure the community and core team have had a chance to review the idea and approve it. The core team has sole responsibility for merging new features based on proposals to `Ansible-core <https://github.com/ansible/ansible>`_.
+
+
+Features, high-level design, and roadmap of Ansible collections
+===============================================================
+
+Collections maintainers define features, high-level design, and roadmap of the collections themselves and are responsible for merging new fearures to `Ansible collections <https://github.com/ansible-collections/>`_ based on proposals discussed with their communities.
 
 Our workflow on GitHub
 ======================
 
-As a committer, you may already know this, but our workflow forms a lot of our team policies. Please ensure you're aware of the following workflow steps:
+As a committer, you may already know this, but our workflow forms a lot of our team policies. Please ensure you are aware of the following workflow steps:
 
 * Fork the repository upon which you want to do some work to your own personal repository
 * Work on the specific branch upon which you need to commit
-* Create a Pull Request back to the Ansible repository and tag the people you would like to review; assign someone as the primary "owner" of your request
-* Adjust code as necessary based on the Comments provided
-* Ask someone on the Core Team to do a final review and merge
+* Create a pull request back to the upstream repository and tag the people you would like to review; assign someone as the primary "owner" of your pull request
+* Adjust code as necessary based on the comments provided
+* Ask someone from the repository committers to do a final review and merge
 
 Addendum to workflow for committers:
 ------------------------------------
 
-The Core Team is aware that this can be a difficult process at times. Sometimes, the team breaks the rules by making direct commits or merging their own PRs. This section is a set of guidelines. If you're changing a comma in a doc, or making a very minor change, you can use your best judgement. This is another trust thing. The process is critical for any major change, but for little things or getting something done quickly, use your best judgement and make sure people on the team are aware of your work.
+The Core Team is aware that this can be a difficult process at times. Sometimes, the team breaks the rules by making direct commits or merging their own pull requests. This section is a set of guidelines. If you are changing a comma in a documentation, or making a very minor change, you can use your best judgement. This is another trust thing. The process is critical for any major change, but for little things or getting something done quickly, use your best judgement and make sure people on the team are aware of your work.
```

Comment:
```suggestion
The Core Team is aware that this can be a difficult process at times. Sometimes, the team breaks the rules by making direct commits or merging their own pull requests. This section is a set of guidelines. If you are changing a comma in documentation, or making a very minor change, you can use your best judgement. This is another trust thing. The process is critical for any major change, but for little things or getting something done quickly, use your best judgement and make sure people on the team are aware of your work.
```